doc/classes/TabContainer.xml

```diff
@@ -30,6 +30,13 @@
 				Returns the previously active tab index.
 			</description>
 		</method>
+		<method name="get_tab_bar" qualifiers="const">
+			<return type="TabBar" />
+			<description>
+				Returns the [TabBar] contained in this container.
+				[b]Warning:[/b] This is a required internal node, removing and freeing it may cause a crash. If you wish to hide it or any of its children, use their [member Window.visible] property.
```

Comment:
This should also warn about editing the tabs themselves. Also, `Window.visible`?

Reply:
Oh, I copied this from the `get_popup` documentation.